pkg/apis/pipeline/v1/taskrun_types.go

```diff
@@ -71,14 +71,14 @@ type TaskRunSpec struct {
 	// This field is only supported when the alpha feature gate is enabled.
 	// +optional
 	// +listType=atomic
-	StepOverrides []TaskRunStepOverride `json:"stepOverrides,omitempty"`
+	StepSpecs []TaskRunStepOverride `json:"stepSpecs,omitempty"`
```

Comment:
Can I also confirm here only the golang name shall be changed not the types as in the single list for `TaskRunStepOverride`?

Reply:
We can rename `TaskRunStepOverride` to `TaskRunStepSpec`